examples/nextjs/src/components/my-page.js

```diff
@@ -1,24 +1,24 @@
-"use client";
+'use client';
 
-import WebPageContent from "./content-types/webPageContent";
-import Banner from "./content-types/banner";
-import Blog from "./content-types/blog";
-import Activity from "./content-types/activity";
-import CallToAction from "./content-types/callToAction";
-import CalendarEvent from "./content-types/calendarEvent";
-import Product from "./content-types/product";
-import ImageComponent from "./content-types/image";
+import WebPageContent from './content-types/webPageContent';
+import Banner from './content-types/banner';
+import Blog from './content-types/blog';
+import Activity from './content-types/activity';
+import CallToAction from './content-types/callToAction';
+import CalendarEvent from './content-types/calendarEvent';
+import Product from './content-types/product';
+import ImageComponent from './content-types/image';
 
-import Header from "./layout/header/header";
-import Footer from "./layout/footer/footer";
-import Navigation from "./layout/navigation";
-import { usePathname, useRouter } from "next/navigation";
-import { DotcmsLayout } from "@dotcms/react";
-import { withExperiments } from "@dotcms/experiments";
-import { CustomNoComponent } from "./content-types/empty";
+import Header from './layout/header/header';
+import Footer from './layout/footer/footer';
+import Navigation from './layout/navigation';
+import { usePathname, useRouter } from 'next/navigation';
+import { DotcmsLayout } from '@dotcms/react';
+import { withExperiments } from '@dotcms/experiments';
+import { CustomNoComponent } from './content-types/empty';
 
-import { usePageAsset } from "../hooks/usePageAsset";
-import NotFound from "@/app/not-found";
+import { usePageAsset } from '../hooks/usePageAsset';
```

Comment:
Why this keep happening?

Reply:
My global prettier is configured for single quotes and maybe someone else in the team uses double quotes in their IDE, the examples does not have prettier